Linux/sgx/libsgx_tsgxssl/tsgxssl_version.cpp

```diff
@@ -30,7 +30,7 @@
  */
 
 // This string should be updated before every release!!
-#define STRFILEVER "1.1.1o"
+#define STRFILEVER "1.1.1s"
```

Comment:
Should be 1.1.1t? Same comment for usgxssl_version.cpp.

Reply:
Yes, it should be. I missed this on the master branch. 
@johnvenn Please correct it. And I will fix it on the master branch.

Reply:
> Yes, it should be. I missed this on the master branch. @johnvenn Please correct it. And I will fix it on the master branch.

fixed in the PR.